bee-block/CHANGELOG.md

```diff
@@ -83,13 +83,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- First implementation of the Message RFC;
+- First implementation of the Block RFC;
```

Comment:
The old changelog shouldn't be modified

Reply:
Yeah, since it's a new crate, I'll reset to 0.1 anyway and remove most stuff from the changelog